src/gam.py

```diff
@@ -755,8 +755,9 @@ def callGAPIpages(service, function, items,
   total_items = 0
   while True:
     this_page = callGAPI(service, function, soft_errors=soft_errors,
-           throw_reasons=throw_reasons, retry_reasons=retry_reasons, **kwargs)
```

Comment:
my changes eliminated the need for a separate pageToken variable keeping it a part of kwargs. Let's stick with that.

Reply:
Right, I didn't change that. The only changes required from the original code are:
drop pageToken = None
drop pageToken=pageToken, in call
add kwargs[u'pageToken'] = pageToken at end
This code performs redundant checks.
```
   if this_page and u'nextPageToken' in this_page:
+      kwargs[u'pageToken'] = this_page[u'nextPageToken']
+    else:

Reply:
Not sure what's going on here then, might be a git issue because the PR as I'm seeing it is adding "pageToken = None" back in....